Documentation/Diagnostics/PH2103.md

```diff
@@ -45,6 +45,17 @@ class GoodExample
 
 ```
 
+## Exceptions
+
+The following types of arguments are exempt from this rule:
+* Invocations of ToString()
+* Invocations within constructor initializers (i.e., `this` and `base`)
+* Invocations to static methods (e.g., `Times.Once`, `Marshal.SizeOf`, `Mock.Of`, etc.)
+* Inner invocations of nested invocations
+* `nameof`
```

Comment:
Also consider to add an exception for: sizeof

Reply:
Unlike `nameof` (which is apparently an invocation), `sizeof` (and `new`) are considered operators, and therefore are not caught by the analyzer. No code changes needed, but added a Unit Test to confirm desired functionality.